projects/packages/external-media/src/class-external-media.php

```diff
@@ -80,11 +80,47 @@ private static function get_data() {
 		}
 
 		return array(
-			'wpcomBlogId'    => $blog_id,
-			'pluginBasePath' => plugins_url( '', Constants::get_constant( 'JETPACK__PLUGIN_FILE' ) ),
-			'ai-assistant'   => array(
+			'wpcomBlogId'         => $blog_id,
+			'pluginBasePath'      => plugins_url( '', Constants::get_constant( 'JETPACK__PLUGIN_FILE' ) ),
+			'ai-assistant'        => array(
 				'is-enabled' => apply_filters( 'jetpack_ai_enabled', $jetpack_ai_enabled ),
 			),
+			'next40pxDefaultSize' => self::site_supports_next_default_size(),
 		);
 	}
+
+	/**
+	 * Check whether the environment supports the newer default size of elements, gradually introduced starting with WP 6.4.
+	 *
+	 * @since 14.0
```

Comment:
Since it was originally in Jetpack, let's indicate that:

```suggestion
	 * @since jetpack-14.0
```